src/OmniSharp.Roslyn.CSharp/Services/Buffer/ChangeBufferService.cs

```diff
@@ -0,0 +1,26 @@
+using System.Composition;
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Buffer
+{
+    [OmniSharpHandler(typeof(RequestHandler<ChangeBufferRequest, object>), LanguageNames.CSharp)]
```

Comment:
Seems wrong to have this twice `typeof(RequestHandler<ChangeBufferRequest, object>)` Can we use the route name here instead? `/changebuffer` etc.